relay-server/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/kubearmor/kubearmor-relay-server/relay-server
 
-go 1.15
+go 1.22
```

Comment:
Did we verify if this is enough? We might need to mention 1.22.1 in toolchain

Reply:
it should be fine, i can build locally without toolchain specified.